twake/backend/core/src/Twake/Discussion/Services/MessageSystem.php

```diff
@@ -580,6 +580,7 @@ public function sendToNode($channel, $message, $did_create){
 
         $title = "";
         $text = $this->buildShortText($message);
+        $body = $text; // we need original body just in case text gets updated further on
```

Comment:
Why do you want the full body ? It can comes to errors with rabbitmq (rabbit only accept small messages) and I don't see the need here ? :)

Reply:
Ah sorrt I just understood:)